k8s/stacks/observ.cue

```diff
@@ -31,5 +32,15 @@ ObservabilityStack: v1.#Stack & {
 				values: {}
             }
         }
+        "pixie": pixie.#PixieChart & {
+			helm: {
+				version: "0.1.6"
+				release: "pixie"
+				values: {
```

Comment:
this prevents the user from overriding the chart's default values, CUE will complain